docs/source/conf.py

```diff
@@ -0,0 +1,57 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import git_machete
+
```

Comment:
Also! will/should this directory be packaged into `sdist`/`bdist_wheel`? 🤔

Reply:
No, it doesn't seem so. This import requires only that the package is installed if we want to build documentation, but does NOT require for the script to be usable to have `docs/*` available. Also this package is installed in `testenv:docs`, so for the purpose of building documentation via `tox -e docs` we should be covered.